content/en/developers/integrations/create-an-integration-detection-rule.md

```diff
@@ -0,0 +1,146 @@
+---
+further_reading:
+- link: https://docs.datadoghq.com/security/cloud_siem/log_detection_rules
+  tag: Documentation
+  text: Log Detection Rules
+kind: documentation
+title: Create an Integration Detection Rule
+description: Learn how to create a detection rule for your integration.
```

Comment:
change to: Learn how to create a Cloud SIEM detection rule for your integration.